src/rules/converters/ban.ts

```diff
@@ -0,0 +1,67 @@
+import { RuleConverter } from "../converter";
+
+type BanRule = {
+    object?: string;
+    property?: string;
+    message?: string;
+};
+
+export const convertBan: RuleConverter = tslintRule => {
+    let bannedObjMethod: BanRule[] = [];
+
+    for (const rule of tslintRule.ruleArguments) {
+        switch (rule.constructor.name) {
+            case "String":
+                bannedObjMethod.push({
+                    property: rule,
+                });
```

Comment:
I think this'll need some work. TSLint's `ban` rule bans both _globals_ and _properties_. ESLint splits those between `no-restricted-globals` and `no-restricted-properties`.

For example, this TSLint configuration:

```json
{
    "rules": {
        "ban": [
            true,
            "eval",
        ]
    }
}
```

...is, from my understanding, equivalent to this ESLint configuration:

```json
{
    "rules": {
        "no-restricted-globals": [
            "error",
            "eval"
        ]
    }
}
```

Reply:
I am confused as the roadmap states that it should be only the `no-restricted-properties` rule. [ESLint rule](https://eslint.org/docs/rules/no-restricted-properties)

Reply:
@KingDarBoja perhaps the roadmap is incorrect?

Reply:
Could be outdated but should be good if someone check that.